src/hardware/input/mouseif_dos_driver.cpp

```diff
@@ -1210,31 +1213,31 @@ static uint8_t update_buttons(const MouseButtons12S new_buttons_12S)
 	};
 
 	uint8_t mask = 0;
-	if (new_buttons_12S.left && !buttons.left) {
+	if (in_buttons_12S.left && !buttons.left) {
 		mark_pressed(0);
 		mask |= static_cast<uint8_t>(MouseEventId::PressedLeft);
-	} else if (!new_buttons_12S.left && buttons.left) {
+	} else if (!in_buttons_12S.left && buttons.left) {
 		mark_released(0);
 		mask |= static_cast<uint8_t>(MouseEventId::ReleasedLeft);
 	}
 
-	if (new_buttons_12S.right && !buttons.right) {
+	if (in_buttons_12S.right && !buttons.right) {
 		mark_pressed(1);
 		mask |= static_cast<uint8_t>(MouseEventId::PressedRight);
-	} else if (!new_buttons_12S.right && buttons.right) {
+	} else if (!in_buttons_12S.right && buttons.right) {
 		mark_released(1);
 		mask |= static_cast<uint8_t>(MouseEventId::ReleasedRight);
 	}
 
-	if (new_buttons_12S.middle && !buttons.middle) {
+	if (in_buttons_12S.middle && !buttons.middle) {
 		mark_pressed(2);
 		mask |= static_cast<uint8_t>(MouseEventId::PressedMiddle);
-	} else if (!new_buttons_12S.middle && buttons.middle) {
+	} else if (!in_buttons_12S.middle && buttons.middle) {
 		mark_released(2);
 		mask |= static_cast<uint8_t>(MouseEventId::ReleasedMiddle);
 	}
 
-	buttons = new_buttons_12S;
+	buttons = in_buttons_12S;
```

Comment:
Yeah, `new` was a lot better IMO.

Reply:
Changed to _new, everywhere. I have changed this to `in_` to adapt to coding style I found in other places - but if you think `new_` is better, fine for me.